.CI/create_feedstocks.py

```diff
@@ -126,6 +157,19 @@ def write_token(name, token):
             out = subprocess.check_output(['git', 'push', 'upstream_with_token', 'master'], cwd=feedstock_dir,
                                           stderr=subprocess.STDOUT)
 
+            # Add team members as maintainers.
+            if conda_forge:
+                meta = MetaData(recipe_dir)
+                maintainers = meta.meta.get('extra', {}).get('recipe-maintainers', [])
+                team = create_team(
+                    conda_forge,
+                    name.lower(),
+                    'The {} {} contributors!'.format(choice(superlative), name),
+                    repo_names=['conda-forge/{}'.format(os.path.basename(feedstock_dir))]
+                )
+                for each_maintainer in maintainers:
+                    team.add_membership(each_maintainer)
```

Comment:
It looks like the other script does 
```requestJsonAndCheck(
                                   "PUT",
                                    team.url + "/memberships/" + new_member)```

Might be worth checking that pygithub doesn't change the default arguments, but otherwise, sounds sensible to me.